src/locale/en.ts

```diff
@@ -0,0 +1,741 @@
+import { Industry } from "../types";
+
+const en: Industry[] = [
```

Comment:
hello @mastepanoski, I was thinking of keeping the translations in the same file, so that is it is easier to go through and we do not have to keep duplicates.

what i mean is this, how about we we keep the translated names in the same object:

```json
{
    "id": 1,
    "linkedinId": 47,
    "name": "Accounting",
    "translations": {
      "es": "es-name"
    }
  }
  ```
  
  so we should keep a single json file containing this, then write a script, that creates each languages translations during build stage.
  
  what do you think?

Reply:
This is a good idea if the script puts the translation for each language in separate files, so that  only the desired language can be loaded in memory. But if you are going to include 10 or more languages, a singe large JSON file might be unwieldy. An alternative is to use the English version as default which contains all the attributes (id, linkedinId, name), and each translation only contains (id, name) or (defaultName, translation), then when a language is loaded, the loader could merge the new language with the default language, it might be useful for partial translation (for example, if you add a new industry category, all other languages might be updated  progressively). 

Another option is to create separate files with translations, each containing an array of translations [name1, name2, ...], and the position of the array matches the industry ID. This option allows faster translation using Google Translate or an automated tool, and is easy to merge with the default structure.

Reply:
yea, the idea is for the script to put the translation for each language in separate files, yea, that is better, so you only import what you need.

Reply:
if you agree with this approach, then we can merge this in, and create another PR?

Reply:
Ok, let's do it.